services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/HankeControllerTest.kt

```diff
@@ -46,21 +48,21 @@ class HankeControllerTest {
         Mockito.`when`(hankeService.loadHanke(mockedHankeTunnus))
                 .thenReturn(Hanke(1234, mockedHankeTunnus, true,
                         "Mannerheimintien remontti remonttinen", "Lorem ipsum dolor sit amet...",
-                        getCurrentTimeUTC(), getCurrentTimeUTC(), Vaihe.OHJELMOINTI, null,
+                        getDatetimeAlku(), getDatetimeLoppu(), Vaihe.OHJELMOINTI, null,
                         1, "Risto", getCurrentTimeUTC(), null, null, SaveType.DRAFT))
 
         val response: ResponseEntity<Any> = hankeController.getHankeByTunnus(mockedHankeTunnus)
 
         Assertions.assertThat(response.statusCode).isEqualTo(HttpStatus.OK)
         Assertions.assertThat(response.body).isNotNull
-        Assertions.assertThat((response.body as Hanke).nimi).isNotEmpty()
+        Assertions.assertThat((response.body as Hanke).nimi).isNotEmpty
```

Comment:
This gives an error in my environment. Maybe ( ) should be returned to avoid problems in other environments too?

Reply:
Returning () with a comment so that they don't get accidentally removed again.